<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Abandoned Farmhouse Adventure</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout4">
        <item>
         <widget class="QFrame" name="moveFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="directionsGroupBox">
             <property name="title">
              <string>Move Commands:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="1">
               <widget class="QPushButton" name="eastButton">
                <property name="text">
                 <string>E&amp;ast</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="westButton">
                <property name="text">
                 <string>&amp;West</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="downButton">
                <property name="text">
                 <string>D&amp;own</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="northButton">
                <property name="text">
                 <string>&amp;North</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="upButton">
                <property name="text">
                 <string>U&amp;p</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="southButton">
                <property name="text">
                 <string>&amp;South</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="globalFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="commandsGroupBox">
             <property name="title">
              <string>Global Commands:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="lookButton">
                <property name="text">
                 <string>&amp;Look</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="inventoryButton">
                <property name="text">
                 <string>&amp;Inventory</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="helpButton">
                <property name="text">
                 <string>&amp;Help</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="quitButton">
                <property name="text">
                 <string>&amp;Quit</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox2">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Item Commands:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="takeButton">
              <property name="text">
               <string>&amp;Take</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="dropButton">
              <property name="text">
               <string>&amp;Drop</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="useButton">
              <property name="text">
               <string>&amp;Use</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="examineButton">
              <property name="text">
               <string>&amp;Examine</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="inventoryGroupBox">
           <property name="title">
            <string>Inventory Items:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="2">
             <widget class="QPushButton" name="inventoryButton3">
              <property name="text">
               <string>flashlight</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="inventoryButton2">
              <property name="text">
               <string>oil</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="inventoryButton4">
              <property name="text">
               <string>candy bar</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="inventoryButton5">
              <property name="text">
               <string>book</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="inventoryButton1">
              <property name="text">
               <string>lamp</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Local Items:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QPushButton" name="objectButton1">
              <property name="text">
               <string>bookcase</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="objectButton5">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="objectButton2">
              <property name="text">
               <string>old radio</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="objectButton4">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="objectButton3">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="objectButton6">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox3">
      <property name="title">
       <string>Status:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="documentTitle">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>upButton</tabstop>
  <tabstop>northButton</tabstop>
  <tabstop>westButton</tabstop>
  <tabstop>eastButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>southButton</tabstop>
  <tabstop>takeButton</tabstop>
  <tabstop>dropButton</tabstop>
  <tabstop>useButton</tabstop>
  <tabstop>examineButton</tabstop>
  <tabstop>inventoryButton1</tabstop>
  <tabstop>inventoryButton2</tabstop>
  <tabstop>inventoryButton3</tabstop>
  <tabstop>inventoryButton4</tabstop>
  <tabstop>inventoryButton5</tabstop>
  <tabstop>lookButton</tabstop>
  <tabstop>inventoryButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>objectButton1</tabstop>
  <tabstop>objectButton2</tabstop>
  <tabstop>objectButton3</tabstop>
  <tabstop>objectButton4</tabstop>
  <tabstop>objectButton5</tabstop>
  <tabstop>objectButton6</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
